range.c

```diff
@@ -998,12 +1002,16 @@ range_begin(VALUE range)
 
 /*
  *  call-seq:
- *     rng.end    -> obj
+ *    end -> object
```

Comment:
Likewise, `self.end` may be better than `end`

Reply:
Fixed.